### DISCUSSION TITLE
HUGE Thank You to Valnoxy - T480 with Ventura 13.3 Success!

### DISCUSSION THREAD

**Discussion by wifihifi** [valnoxy/t480-oc#87] [Show and tell]
It took a couple of hours to setup the installer - really it was figuring out the mac address that threw me but a bit of googling about and I figured out what the first 4 digits needed to be.  Absolutely thrilled with the result! I'm running Ventura 13.3, with a successful sign-in to my icloud account (I still can't believe it works so well).

With my T480 I started the install and walked away for a couple of hours, when I returned the OS was ready and waiting.  

Using my Lenovo USB-C Dock with display link drivers and all is well, Mac USB keyboard - all buttons mapped correctly with no fiddling.  Wi-fi works great, so far all my general day-to-day apps work great too.  I'm a general kind of every day mac user so not taxing this in the least. 

THANK YOU THANK YOU THANK YOU!!
![Screenshot 2023-04-02 at 5 35 08 PM](https://user-images.githubusercontent.com/129695360/229380614-67ebade9-c003-4229-8423-0f2dd76aa7f7.png)

**Comment by seargentpepper** [comment]
How it's your Sleep-Wake? Mine it's stuck on Ventura, if it sleeps it's condemned to force reboot as screen is not coming back from black.
